code/socialDistribution/templates/tagtemplates/author.html

```diff
@@ -57,7 +57,10 @@ <h6 class="card-subtitle mb-2 text-muted px-1">
                         <div class="author-card-opt d-flex ml-10">
                             <form method="GET" action="{% url 'socialDistribution:inbox' %}">
                                 {% csrf_token %}
-                                <button class="btn bg-white text-primary px-1">See inbox</button>
+                                <button class="btn bg-blue text-white btn-md btn-primary rounded-pill">See inbox</button>
+                            </form>
+                            <form method="GET" action="{% url 'socialDistribution:unlisted-posts' %}">
```

Comment:
consider adding a bit of spacing: 
something like padding-left: 1em;

Reply:
I actually meant padding on the form tag so that there is spacing between the buttons. Maybe add it to the pr for publicly available unlisted posts. thanks.
![Screen Shot 2021-11-29 at 5 59 19 PM](https://user-images.githubusercontent.com/44795394/143966502-bbb2d091-574b-4511-99d8-3ca285c80ed3.png)
![Screen Shot 2021-11-29 at 5 58 05 PM](https://user-images.githubusercontent.com/44795394/143966505-4ecd39df-3ae7-4b40-ba24-97749f572cd5.png)